Bottom: d24b2b5392b102194b4cf8c9aaf99c432fe68238
Top:    9a295dcf4ad7975e93f19a03f7b5df841e862bda
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 22:40:37 +0000

Refresh of attempt-to-cache-emission-of

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index f40613c..a21068d 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -417,46 +417,8 @@ object3d_from_contours (POLYAREA *contours,
         }
 #else
 #endif
-    object_appearance = master_object_appearance;
-    top_bot_appearance = master_top_bot_appearance;
-
-    object3d_set_appearance (object, object_appearance);
-
-    vertices = malloc (sizeof (vertex3d *) * 2 * npoints); /* (n-bottom, n-top) */
-    edges    = malloc (sizeof (edge_ref  ) * 3 * npoints); /* (n-bottom, n-top, n-sides) */
-    faces    = malloc (sizeof (face3d *) * (npoints + 2)); /* (n-sides, 1-bottom, 1-top */
-
-    /* Define the vertices */
-    ct = contour;
-    offset_in_ct = 0;
-    ct_npoints = get_contour_npoints (ct);
->>>>>>> patched
-
-      contour = pa->contours;
-      ncontours = 0;
-      npoints = 0;
-
-      ct = contour;
-      while (ct != NULL)
-        {
-          ncontours ++;
-          npoints += get_contour_npoints (ct);
-          ct = ct->next;
-        }
-
-      object = make_object3d (PCB->Name);
-
-      if (master_object_appearance != NULL)
-        {
-          object_appearance = make_appearance ();
-          appearance_set_appearance (object_appearance, master_object_appearance);
-        }
-
-      if (master_top_bot_appearance != NULL)
-        {
-          top_bot_appearance = make_appearance ();
-          appearance_set_appearance (top_bot_appearance, master_top_bot_appearance);
-        }
+      object_appearance = master_object_appearance;
+      top_bot_appearance = master_top_bot_appearance;
 
       object3d_set_appearance (object, object_appearance);
 
diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index d35d863..2d78563 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -466,11 +466,7 @@ object3d_list_export_to_step_part (GList *objects, const char *filename)
       if (multiple_bodies)
         g_string_append_printf (body_name, " - %i", part);
 
-<<<<<<< current
-      object3d_to_step_body_fragment (step, object, body_name->str, &comp_brep, &styled_items);
-=======
-    object3d_to_step_body_fragment (step, object, body_name->str, &comp_brep, &styled_items, appear_hash);
->>>>>>> patched
+      object3d_to_step_body_fragment (step, object, body_name->str, &comp_brep, &styled_items, appear_hash);
 
       g_string_free (body_name, true);
 
@@ -553,13 +549,8 @@ object3d_list_export_to_step_assy (GList *objects, const char *filename)
           g_string_append_printf (body_name, " - %i", part);
         }
 
-<<<<<<< current
       object3d_to_step_fragment (step, object, part_id->str, part_name->str, "PCB model", body_name->str,
-                                 &comp_shape_definition_representation, &comp_placement_axis);
-=======
-    object3d_to_step_fragment (step, object, part_id->str, part_name->str, "PCB model", body_name->str,
-                               &comp_shape_definition_representation, &comp_placement_axis, appear_hash);
->>>>>>> patched
+                                 &comp_shape_definition_representation, &comp_placement_axis, appear_hash);
 
       g_string_free (part_id, true);
       g_string_free (part_name, true);
